tests/cql_query_test.cc

```diff
@@ -2741,28 +2741,25 @@ SEASTAR_TEST_CASE(test_alter_table) {
             return e.execute_cql("alter table tat with comment = 'This is a comment.';").discard_result();
         }).then([&e] {
             BOOST_REQUIRE_EQUAL(e.local_db().find_schema("ks", "tat")->comment(), sstring("This is a comment."));
-            return e.execute_cql("alter table tat alter r2 type blob;").discard_result();
-        }).then([&e] {
             return e.execute_cql("select pk1, c1, ck2, r1, r2 from tat;");
         }).then([&e] (shared_ptr<cql_transport::messages::result_message> msg) {
             assert_that(msg).is_rows().with_rows({
                 { int32_type->decompose(1), int32_type->decompose(2), int32_type->decompose(3), int32_type->decompose(4), int32_type->decompose(5) },
             });
         }).then([&e] {
-            return e.execute_cql("insert into tat (pk1, c1, ck2, r2) values (1, 2, 3, 0x1234567812345678);").discard_result();
+            return e.execute_cql("insert into tat (pk1, c1, ck2, r2) values (1, 2, 3, 4);").discard_result();
```

Comment:
I think the point of these lines of  test code was to change the type of this column, and then write new data with the new type and then read that data, and see that it all works. If you don't do this change, what's the point of writing (1,2,3,4) into the data? Didn't we have exactly this data from what the previous lines in the test did? So I think this line here, and the lines below testing the result of the SELECT, can be removed.